src/pages/explore/dex/_components/ExploreDTokensSection.tsx

```diff
@@ -0,0 +1,215 @@
+  }, [tokenSectionChoice]);
+
+  const dimension = useWindowDimensions();
+
+  return (
+    <Container className="mb-[240px]">
+      <div className="flex flex-col md:items-center items-start">
+        <SectionTitle text={t("exploreDTokensSection.title")} />
+        <div
+          className={classNames(
+            "lg:text-[60px] lg:leading-[72px] mt-5 lg:tracking-normal",
+            "text-[40px] leading-[44px] tracking-[-0.02em]"
+          )}
+        >
+          {t("exploreDTokensSection.subtitle")}
+        </div>
+
+        <ExploreDTokensTabs
+          tokenSectionChoice={tokenSectionChoice}
+          setTokenSectionChoice={setTokenSectionChoice}
+        />
+        <ExploreDTokensGrid selectionTokens={selectedTabTokens} />
+
+        <SecondaryButton
+          href="https://defiscan.live/tokens"
+          className="place-self-center md:mt-16 mt-14 lg:py-4 py-3 md:px-[56px] px-[43px] w-fit lg:text-base text-sm lg::font-bold font-semibold"
+          text={t("exploreDTokensSection.secondaryButtonText")}
+          hasDiagonalArrow
+          arrowSize={dimension.width >= 1024 ? 24 : 20}
+        />
+
+        <div className="text-dark-700 md:text-base text-sm place-self-center mt-4 text-center font-desc">
+          {t("exploreDTokensSection.pricesDesc")}
+        </div>
+      </div>
+    </Container>
+  );
+}
+
+const crypto = ["DUSD", "BTC", "ETH", "USDC", "EUROC", "USDT", "LTC", "BCH"];
+const dStocks = [
+  "TSLA",
+  "BABA",
+  "GOOGL",
+  "AAPL",
+  "AMZN",
+  "NVDA",
+  "MSFT",
+  "PYPL",
+];
+const etfs = ["ARKK", "SPY", "GLD", "VNQ", "VOO", "SLV", "URTH", "QQQ"];
+
+function reorderObject(obj, order) {
+  const newObj = {};
+  order.forEach((key) => {
+    if (Object.hasOwn(obj, key)) {
+      newObj[key] = obj[key];
+    }
+  });
+  return newObj;
+}
+
+function setTokens(
+  tokenNameMap: { [symbol: string]: string },
+  allUncleanTokens: DexPricesResult,
+  setCryptoTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setDstocksTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setEtfsTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >,
+  setSelectedTabTokens: Dispatch<
+    SetStateAction<{
+      [symbol: string]: DexPrice;
+    }>
+  >
+) {
+  // eslint-disable-next-line no-param-reassign
+  allUncleanTokens.dexPrices.USDT = {
+    token: {
+      id: "1",
+      name: "Tether USD",
+      symbol: "USDT",
+      displaySymbol: "dUSDT",
+    },
+    denominationPrice: "1.00",
+  };
+
+  const allTokensWithUpdatedName = {};
+
+  // eslint-disable-next-line array-callback-return
+  Object.entries(allUncleanTokens.dexPrices).map(([key, value]) => {
+    allTokensWithUpdatedName[key] = value;
+    allTokensWithUpdatedName[key].token.name =
+      tokenNameMap[value.token.symbol] ?? value.token.name;
+  });
+
+  const initialSelectedTabTokens = {};
+  // eslint-disable-next-line array-callback-return
```

Comment:
if you dont need to `return`, consider `forEach()`

Reply:
Yes I want